### ISSUE TITLE
Machine vars are sent four times via BCP when monitoring starts

### ISSUE THREAD

**Issue by arthurkehrwald** [missionpinball/mpf#1846]
When a BCP server requests to start monitoring machine variables, MPF sends the current value of every machine variable... four times. One way to see this is by using the [Unity BCP server](https://github.com/missionpinball/unity-bcp-server) and looking at the incoming messages in the console.

**Comment by avanwinkle** [comment]
Yes, this was a known issue that was fixed for 0.80 but hasn't been back ported to 0.57 yet. I'll prioritize that!
